test/utils/math/SignedMath.test.js

```diff
@@ -1,94 +1,51 @@
-const { BN, constants } = require('@openzeppelin/test-helpers');
+const { ethers } = require('hardhat');
 const { expect } = require('chai');
-const { MIN_INT256, MAX_INT256 } = constants;
+const { loadFixture } = require('@nomicfoundation/hardhat-network-helpers');
+const { min, max } = require('../../helpers/math');
 
-const SignedMath = artifacts.require('$SignedMath');
+async function testCommutative(fn, lhs, rhs, expected, ...extra) {
+  expect(await fn(lhs, rhs, ...extra)).to.deep.equal(expected);
+  expect(await fn(rhs, lhs, ...extra)).to.deep.equal(expected);
+}
```

Comment:
What do you think of moving this function to `helpers/math.js`?

Reply:
Maybe. I'm not sure if that is math specific or if it should go in another helper.

Reply:
I think "commutative" is a math property so it can go in `math.js` imo. An alternative would be just `methods.js`, but feel free to decide, I just don't want to keep it twice

Reply:
We discussed it during a meeting and agreed to keep duplicates for now. We may reconsider after finishing the tests migration and consolidating the helpers at the end.

Solving